### PR TITLE
Added magnifying glass image on search button.

### LINKED ISSUE #192: Replace search button with magnifying glass
Replace the existing search button with a little magnifying glass button beside the keyword search box

Acceptance Criteria
- [ ] There is a magnifying glass button beside the keyword search box
- [ ] The existing search button is removed

### PR DESCRIPTION
**Pull request by JacOng17**
Added magnifying glass image on search button.

Closes #192

### CODE REVIEW

File: src/assets/scss/components/_search.scss [hunk 1]
```diff
@@ -9,4 +9,36 @@
       margin: .25rem .5rem .25rem 0;
     }
   }
+
+  @media only screen and (max-width: 767px) {
+    &__topic {
+      position: relative;
+    }
+
+    &__magnifyingGlass {
```
Comment (dlouise64) [inline src/assets/scss/components/_search.scss]: @JacOng17 Is camel casing used anywhere else in the code? If not we should follow the convention ie magnifying-glass (if thats how it is)

File: src/assets/scss/components/_search.scss [hunk 2]
```diff
@@ -9,4 +9,36 @@
...
+      background-image: url('../img/magnifying-glass.png');
+      background-size: cover;
+      height: 1.75rem;
+      width: 1.75rem;
+      position: absolute;
+      bottom: -.25rem;
+      padding-left: 0;
```
Comment (dlouise64) [inline src/assets/scss/components/_search.scss]: @JacOng17 notice how weve duplicated the styles but only added padding-left to both `&__magnifyingGlass` 
We can use `@extend` for that. Take a look here https://sass-lang.com/documentation/file.SASS_REFERENCE.html#extend

File: src/assets/scss/components/_search.scss (hunk 1 above)
Comment (JacOng17) [reply src/assets/scss/components/_search.scss]: Not too sure, will have a look if not will correct it :+1:

File: src/components/search-form.js [hunk 3]
```diff
@@ -65,12 +65,11 @@ export default class SearchForm extends Component {
               </div> )
               : null
             }
-            <div>
+            <div className="search__topic">
```
Comment (Br3nda) [inline src/components/search-form.js]: does the double underscore mean something?

Comment (dlouise64) [reply src/components/search-form.js]: @Br3nda yup this is following BEM methodology.  double underscores is used when you want to specify the element name. `.search` would be the block (B), `__topic` is the element (E) and M is for Modifier. We use `--` for modifiers.

Examples:

```
.search

.search__topic

.search__topic--error
```

Comment (Br3nda) [reply src/components/search-form.js]: Thank! i learned something

Comment (lamorrison) [reply src/components/search-form.js]: ✨

Comment (JacOng17) [reply src/components/search-form.js]: Linked to &__topic { '_search.scss`

File: src/assets/scss/components/_search.scss (hunk 2 above)
Comment (dlouise64) [reply src/assets/scss/components/_search.scss]: @JacOng17 Just as reminder as we discovered, we can't use @extend within @media but we can use @mixin :)

**Comment by dlouise64** [comment]
✨ awesome :)

**Pull request merged**